src/ui/public/vislib/visualizations/pie_chart.js

```diff
@@ -119,6 +119,7 @@ export default function PieChartFactory(Private) {
         return Math.max(0, Math.min(2 * Math.PI, x(d.x)));
       })
       .endAngle(function (d) {
+        if (d.dx < 0.00000001) return x(d.x);
```

Comment:
fyi; in graphics/trig libraries, you'll generally see these type of epsilons written in scientific number notation. so `1e-8`, it looks less googely ;)